opm/grid/cornerpoint_grid.c

```diff
@@ -170,6 +171,7 @@ create_grid_cornerpoint(const struct grdecl *in,
     struct UnstructuredGrid *g;
     int                      ok;
     struct processed_grid    pg;
+    memset(&pg, 0, sizeof(struct processed_grid));
```

Comment:
I think you can get the same effect, and portably, by declaring `pg` as
```C
struct processed_grid pg {0};
```
instead.  That syntax, while strange if you've not seen it before, works very similarly to the pair of empty braces in C++.  It sets the struct's integers to zero, its floating-point members to 0.0, and its pointers to `NULL`.

Is there a quick way to test that in a static analysis context?